welcome/src/main/kotlin/edu/artic/welcome/WelcomeEventsAdapter.kt

```diff
@@ -14,8 +15,8 @@ import kotlinx.android.synthetic.main.welcome_event_cell_layout.view.*
  * @author Sameer Dhakal (Fuzz)
  */
 
-class WelcomeEventsAdapter : AutoHolderRecyclerViewAdapter<WelcomeEventCellViewModel>() {
-    override fun View.onBindView(item: WelcomeEventCellViewModel, position: Int) {
+class WelcomeEventsAdapter : AutoHolderRecyclerViewAdapter<EventCellViewModel>() {
```

Comment:
since you are using the EventCellViewModel at this point and the cells are the same between both. Can we swap over to the EventsAdapter in the events package? The only difference is items and thats control in the WelcomeViewModel

Reply:
That change will need to be covered as a separate part of #298, so I'll close this particular comment as resolved within the scope of #291.